### DISCUSSION TITLE
Unable to execute the 2-run, 3-run_vr bat files

### DISCUSSION THREAD

**Discussion by 25011908vardhan** [harfang3d/car-simulator#16] [Q&A] [answered]
I wanted to try this car-simulator and followed the steps mentioned in Readme, but I am unable to run the 2-run.bat an 3-run_vr.bat files.
Can you please tell me a way to fix this? 
I am attaching the screenshots of the errors in output screen of cmd. 

**For 2-run.bat file ->** 

![2-run](https://user-images.githubusercontent.com/75671830/203306894-97e71605-62bf-4d5d-861d-2f595b3ecdc6.jpeg)

**For 3-run_vr.bat file->**

![3-run_vr](https://user-images.githubusercontent.com/75671830/203307129-4b7ee507-d64e-4a6f-bb12-bd38941cb038.jpeg)

**Comment by disketflu** [comment, marked as answer]
Hello Sir, thanks for the feeback.
The public version of Harfang currently doesn't support running the app in it's current state.
I will be preparing a release right now with the correct binaries so that you can run it and modify it if needed.
I will post a new comment once it is ready as well.

Sincerely,
Clément.

**Reply by 25011908vardhan** [reply]
Thanks for the reply. 
Waiting for the release eagerly.

**Reply by disketflu** [reply]
https://github.com/harfang3d/car-simulator/releases/tag/v1.0.1-hg3.2.4

Here is the release, everything should work right away when running the "2-run.bat" file.

The "3-run_vr.bat" should be executed with SteamVR installed and launched, and a VR headset plugged in otherwise the hg.OpenVrInit() function will fail to execute.

Best Regards,
Clément.

**Reply by 25011908vardhan** [reply]
Thanks a lot!
